Lib/asyncio/queues.py

```diff
@@ -30,6 +30,20 @@ class QueueShutDown(Exception):
     pass
 
 
+class _AsyncQueueIterator:
```

Comment:
I'm curious why you created a new class here as iterator instead of adding ``__anext__`` to queue itself?

Reply:
From https://github.com/python/cpython/issues/119154#issuecomment-2119554265:

> it feels more reasonable to describe queues as aiterables than aiterators.